examples/hotrod/pkg/pool/pool_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pool
+
+import (
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+)
+
+var n int32 = 0
+var wg sync.WaitGroup
+
+func testJob() {
+	atomic.AddInt32(&n, 1)
+	time.Sleep(time.Second)
```

Comment:
why do you need sleep at all, if you're using waitGroup to signal completion?

Reply:
Because i rely on runtime to determine if it is running correctly, sleeping for 1 second makes it easier to calculate the running time.

Reply:
I don't follow. WaitGroup allows you to know that a job was executed, using sleep in addition to that does not add any extra benefits. Making tests relying on the timestamps always makes them unstable in CI where scheduling and CPU allocations are unpredictable.

Reply:
OK, I will remove judging the number of workers by runtime